frontend/src/components/dashboard/page_timings/PageTimingsTable.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+    <VContainer
+        p-0
+        class="users-table"
+    >
+        <VRow
+            class="header my-3"
+            fluid
+        >
+            <VCol>
+                <VSelect
+                    class="option-select"
+                    prefix="Show"
+                    :items="options"
+                    flat
+                    v-model="selected"
+                    @change="changeSelect"
+                />
+            </VCol>
+            <VCol>
+                {{ option }}
+            </VCol>
+        </VRow>
+        <VDataTable
+            class="caption"
+            hide-default-footer
+            hide-default-header
+            :headers="headers"
+            :items="items"
+        />
+    </VContainer>
+</template>
+
+<script>
+    export default {
+        name: 'GroupedTable',
+        props: {
+            items: {
+                type: Array,
+                required: true
+            },
+            option: {
+                type: String,
+                required: true
+            }
+        },
+        data () {
+            return {
+                selected: 'browser',
```

Comment:
it seems this local state is redundant, since you need to get current selected group value from Vuex